tests/integrationtest/l4_crd_test.go

```diff
@@ -1108,3 +1119,418 @@ func TestSharedVIPSvcWithL4Rule(t *testing.T) {
 	TearDownTestForSharedVIPSvcLB(t, g)
 	TeardownL4Rule(t, L4RuleName, NAMESPACE)
 }
+
+func TestCreateDeleteL4RuleSSLCustomValues(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	// setting license to enterprise
+	SetupLicense := func(license string) {
```

Comment:
this function seems to be repeated in two UTs. It will be better if we can move it as a common function.